Dockerfile

```diff
@@ -43,6 +43,13 @@ ENV NODE_ENV=production
 ARG DATABASE_URL
 ENV DATABASE_URL=$DATABASE_URL
 
+# Datadog tracing configuration (can be overridden at runtime)
+ENV SERVICE_NAME=vaults-backend
```

Comment:
Hum, we need to disintguish beween claimer, indexer and server...

Reply:
Yes this is just a default value that can be changed through env var